### PR TITLE
[Upcoming] Release 1.6.0

### PR DESCRIPTION
**Pull request by skoudoro**
Preparation for 1.6.0 release, targeting ~~August 1st~~  January 13-14. 

# Desired PR / issues (need review / feedback / update)

There is room for the following PR to be finished and merge

- [ ] ~~#2599  (Submitter: @frheault)~~
- [ ] ~~#2590 (Submitter: @StongeEtienne)~~
- [ ] #2575 (Submitter: @skoudoro)
- [ ] #2574 (Submitter: )
- [x] #2572 (Submitter: @skoudoro via #2629 )
- [x] #2564 (Submitter: @skoudoro via #2630 )
- [x] #2559 (Submitter: @skoudoro via #2631 )
- [x] #2369 (Submitter: @skoudoro )
- [x]  #2553 (Submitter: @drombas )

Most of them can be done on time, I think :smile:

Please comment to flag any other issues or PR that should be addressed for this release

# Release checklist
- [x] add release_notes 1.6.0
- [x] Update changelog and api_changes
- [x] Update .mailmap
- [ ] Set release number in info.py
- [x] Check the copyright years in doc/conf.py and LICENSE
- [x] Check documentation
- [x] Check Tutorial
- [ ] Check unit test and USE-PRE
- [x] Update index.rst
- [x] Update Highligth.rst, old_news.rst, stateofheart.rst
- [x] Update developers.rst list
- [ ] Update website
- [x] Update deprecation cycle remove long deprecation

### CODE REVIEW

**Comment by drombas** [comment]
Hi @skoudoro!, any chances of adding #2553 to this release?

**Comment by skoudoro** [comment]
> any chances of adding https://github.com/dipy/dipy/pull/2553 to this release?

ok, will look into it @drombas

**Comment by arnaudbore** [comment]
Any update on the release date ? @skoudoro

**Comment by arnaudbore** [comment]
Thank you for your quick answer. Do you have time line ? :)

**Comment by skoudoro** [comment]
@arokem, @Garyfallidis, Can you have a look at this PR? Specially, if you could re-phrase the highlight, it would be great!

Concerning the CI's, I do not know what is going on since I merged the last PR from @arokem. it is frozen, that is strange.... I will try to restart them a bit later.

File: README.rst [hunk 1]
```diff
@@ -37,8 +37,8 @@ Current information can always be found from the DIPY website - http://dipy.org
 Mailing Lists
 =============
 
-Please see the developers' list at
-https://mail.python.org/mailman/listinfo/neuroimaging
+Please see the DIPY community' list at
```
Comment (arokem) [inline README.rst:40]:
```suggestion
Please see the DIPY community list at
```

Comment (arokem) [reply README.rst:40]: Could also be "DIPY community's list", but I prefer no genitive.

File: doc/index.rst [hunk 2]
```diff
@@ -16,31 +16,29 @@ perfusion and structural imaging.
...
-- New reconstruction model added: Residual block Deep Neural Network (ResDNN).
-- Masking management in Affine Registration added.
-- Multiple Workflows updated (DTIFlow, DKIFlow, ImageRegistrationFlow) and added (MotionCorrectionFlow).
-- Compatibility with Python 3.10 added.
-- Migrations from Azure Pipeline to Github Actions.
+**DIPY 1.6.0** is now available. New features include:
+
+- NF: Unbiased groupwise linear bundle registration added.
+- NF: MAP+ constraints added.
+- Generalized PCA to less than 3 spatial dimsAdd positivity constraints to QTI
```
Comment (arokem) [inline doc/index.rst:23]:
```suggestion
- Generalized PCA to less than 3 spatial dims.
- Add positivity constraints to QTI.
```

**Review by arokem — COMMENTED** [review]
Looks good! 🚀

**Pull request merged**